src/test/ks-installer/Chart.yaml

```diff
@@ -1,13 +1,26 @@
-appVersion: 2.1.1
-apiVersion: v1
+apiVersion: v2
+name: ks-installer
 description: The helm chart of KubeSphere, supports installing KubeSphere on existing Kubernetes.
 icon: https://kubesphere.io/static/logo.16097928.svg
-name: ks-installer
-version: 0.1.0
 keywords:
   - kubesphere
   - installer
-home: https://kubesphere.com.cn/
-maintainers:
-  - name: KubeSphere
-    email: kubesphere@yunify.com
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+appVersion: v3.0.0
```

Comment:
`appVersion: 3.0.0` per semantic versioning.

Reply:
OK, I've modified it. Please review again.